content/en/docs/reference/command-line-tools-reference/kube-proxy.md

```diff
@@ -15,6 +15,23 @@ environment variables specifying ports opened by the service proxy. There is an
 addon that provides cluster DNS for these cluster IPs. The user must create a service
 with the apiserver API to configure the proxy.
 
+Use-cases
+
+•	You can make your own proxy with custom behavior (antrea, cilliim, calico? and leverage from the ideas of the implementation of kube-proxy. 
+•	The kube-proxy is only configurable by configmap even though it has several argument options, which can be used for GENERATING a config map.
+o	edit the kube-proxy configmap
+o	restart all kube-proxy pods (i.e. delete them, and the daemonset will have them rescheduled)
+•	The users-space proxy creates a new go routine for every service, this is rarely used, as kernel proxies (which use HNS, IPTables, IPVS, and so on) are much faster and scalable
+
+Descriptions of modes
+
+-	User-space proxy – a deprecated mode of operation in which the Kube-proxy watches for changes to a cluster and will open up a TCP-based port on a Worker Node in the cluster. Iptables is responsible for routing traffic from this worker node’s port to the Kube-proxy. If a pod doesn’t respond, Iptables will retry sending to another pod (note, IPTables is responsible for this, not Kubernetes)
```

Comment:
@jayunit100 by windows side, do you know if user space proxy deals only with TCP protocol? I have no idea in Linux, will have to test